packages/mui-material/src/MenuList/MenuList.js

```diff
@@ -232,6 +232,14 @@ const MenuList = React.forwardRef(function MenuList(props, ref) {
         activeItemIndex = index;
       }
     }
+
+    if (activeItemIndex === index && (child.props.disabled || child.type.muiSkipListHighlight)) {
```

Comment:
```suggestion
    if (activeItemIndex === index && (child.props.disabled || child.props.tabindex < 0)) {
```

Instead of coming up with new prop, should we rather teach developers to use the `tabindex` correctly?

Reply:
That is a very good suggestion that is also completely irrelevant of the list item component being used. I like it lots more than the `muiSkipListHighligh` prop.

Reply:
The MenuList (and dependent compoenents) uses the [roving tabindex pattern](https://www.stefanjudis.com/today-i-learned/roving-tabindex/). Using this pattern, only one of the options (the highlighted one) should have tabindex=0. The rest should have tabindex=-1, so they are not tabbable (navigation between options is done using the arrow keys, not the tab key).

Reply:
Agree, using the tabindex conflicts with the roving tabindex pattern. @michaldudak have you consider using a `data-` attribute instead of static field? Something like `data-mui-select-skip-tab` or something shorter :))